### ISSUE TITLE
fix weapon viewmodels skewing with fov > 85

### ISSUE THREAD

**Issue by OVQ2** [auroramod/iw7-mod#40]
Weapons in Infinite Warfare have a weird skewing effect when at FOVs over 85. I was wondering if it was possible to fix it, and if it was even worth it to fix it. 

Also I have a secondary suggestion that I didn't mention in the issue name being un cheat protect the "gpad_stick_deadzone_min" and "gpad_stick_deadzone_max" commands.

![80fov](https://user-images.githubusercontent.com/115388762/230513475-fd285284-5de4-48c5-99b8-46413518bd42.png)
![120fov](https://user-images.githubusercontent.com/115388762/230513476-8e565c67-d814-4425-adae-9b490817ce94.png)

**Comment by ujicos** [comment]
I think this can be "fixed" by making a custom viewmodel fov command, or rather just not messing with the viewmodel when adjusting FOV, and setting the CG_GUN dvars instead to mitigate it. Im sure devs have a better way.

**Comment by OVQ2** [comment]
Yeah that's kind of what I imagined, but have no idea how they would implement it.
